pages/index.js

```diff
@@ -64,8 +64,8 @@ const NewProducts = ({ products }) => {
                                 <div className="product-container col-md-3 mb-5" key={item.id}>
                                     {/* @TODO need to get rid of using databseId here. */}
                                     <Link
-                                        as={`/product/${item.slug}-${item.databaseId}`}
-                                        href={`/product?slug=${item.slug}-${item.databaseId}`}>
+                                        as={`/product/${item.slug}`}
```

Comment:
In the newer version of next/link we dont need as prop, this will also fix the router break issue when you click on the home icon `WP Decopled` just found that. 
```
href={`/product/${item.slug}-${item.databaseId}`}
```

Reply:
Thanks, done!